src/MCPServer/osdeps/Ubuntu-14.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_comments": [
+    "Add any comments here",
+    "Include here keys, repos, and packages required by this component not already in archivematicaCommon osdeps file"
+  ],
+  "osdeps_repokeys": [
+  ],
+  "osdeps_repos": [
+  ],
+  "osdeps_packages": [
+  { "name": "dbconfig-common", "state": "latest"},
+  { "name": "logapp", "state": "latest"},
+  { "name": "uuid", "state": "latest"}
```

Comment:
I don't think so. Gearman, elasticsearch and mysql are services archivematica relies on, but they should be handled on their own. 

I'm thinking about environments with their own mysql/es servers, were installing the db servers as part of archivematica doesn't make sense.